A rainy day through your window

Drea007 opened the pull request.

shiffman commented:
Great job! You just need to change the image file to have the name "contribution18.png" and then it will show up also!  Without it shows the default challenge thumbnail.

<img width="453" alt="Screen Shot 2023-02-03 at 4 49 49 PM" src="https://user-images.githubusercontent.com/191758/216717553-03472fa0-8ecb-4590-be82-79e7a701a16c.png">

Thank you for your patience in how long it's taken to get these merged! Wonderful work!

If you prefer, I can merge this pull request and rename the file myself if that is easier for you!

Drea007 commented:
I just deleted that previous png file and replaced it by another one with
the "contribution18.png" name 👍🏻

Thank u sm for your assistance, Daniel!

On Fri, Feb 3, 2023, 22:52 Daniel Shiffman ***@***.***> wrote:

> Great job! You just need to change the image file to have the name
> "contribution18.png" and then it will show up also! Without it shows the
> default challenge thumbnail.
>
> [image: Screen Shot 2023-02-03 at 4 49 49 PM]
> <https://user-images.githubusercontent.com/191758/216717553-03472fa0-8ecb-4590-be82-79e7a701a16c.png>
>
> Thank you for your patience in how long it's taken to get these merged!
> Wonderful work!
>
> If you prefer, I can merge this pull request and rename the file myself if
> that is easier for you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodingTrain/thecodingtrain.com/pull/922#issuecomment-1416445562>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUUQKLBD2AZZH7F6O2S5KQLWVV4YNANCNFSM6AAAAAAUQUOBS4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

shiffman commented:
Yay!
